utils/utils.go

```diff
@@ -74,7 +81,15 @@ func GetIntValOrDefault(val int, defaultValue int) int {
 
 // IsSocket returns if the given file is a unix socket.
 func IsSocket(f fs.FileInfo) bool {
```

Comment:
Please add tests for it.

Reply:
PR has tests added to `SocketExists()` to cover the Windows cases, which *should* cover this function, being the only place it's called.